log.go

```diff
@@ -162,23 +161,23 @@ func (l Level) String() string {
 // returns an error if the input string does not match known values.
 func ParseLevel(levelStr string) (Level, error) {
 	switch strings.ToLower(levelStr) {
-	case LevelFieldMarshalFunc(TraceLevel):
+	case strings.ToLower(LevelFieldMarshalFunc(TraceLevel)):
```

Comment:
what about using `EqualFold` instead of double `ToLower`?

Reply:
As far as I'm aware `EqualFold` performs a check (returns a `bool`, so it wouldn't be a drop in replacement for `ToLower`, but we would rewrite the logic with an individual `if`-statement for each case.

Reply:
That’s fine